controller/controller.js

```diff
@@ -0,0 +1,64 @@
+import logger from "../config/logger";
+import Note from "../models/notes";
+
+class Controller {
```

Comment:
Remove all the classes. Replace them with async/await functions.
If we're talking about file name, rename it with `modules/notes/controllers`
Structure:
```
//Here all the routes referenced to Notes, they call controllers
modules/notes/routes/index.js 

//Here all the controllers referenced to Notes, they call DB methods
modules/notes/controllers/getNote.js
modules/notes/controllers/updateNote.js
modules/notes/controllers/deleteNote.js

// Here all DB calls
db/providers/notes.js

//Here we will include note& greetings routers from modules/notes/routes and modules/greeting/routes and start the server
app.js
```